src/main/java/uk/gov/hmcts/reform/sscs/domain/notify/NotificationEventType.java

```diff
@@ -32,6 +32,7 @@
     VIEW_ISSUED("decision_issued", false, false, true, false, false),
     DECISION_ISSUED_2("corDecision", false, false, true, false, false),
     STRUCK_OUT("struckOut", true, true, false, true, false),
+    INTERLOC_VALID_APPEAL("interlocValidAppeal", true, true, true, true, false),
```

Comment:
You sure this should be allowed out of hours?

Reply:
Looks like the Struck out one above is incorrect as well - I know it's not been changed as part of this PR but maybe this can be updated at the same time?